.github/workflows/sync-main.yml

```diff
@@ -0,0 +1,48 @@
+name: automatic sync main branch from llvm-project to llvm
+
+on:
+  schedule:
+  - cron: '*/10 * * * *'
```

Comment:
I think we can expand it to 60 minutes, to reduce load on github runners.

Reply:
How long does it take to run the sync? I would expect something between a few seconds to a minute.
If it's true, 10 minutes sounds reasonable to me.